### ISSUE TITLE
different accuracy under different batchsize in inference

### ISSUE THREAD

**Issue by Mulan337287** [Canjie-Luo/MORAN_v2#86]
I have trained only ASRN model with SynthText, and test in ICDAR2015-2077, but I got the different accuracy if dataloader has different batchsize, 
batch_size=16, acc=61.964, batch_size=1, acc=61.820

**Comment by Canjie-Luo** [comment]
Can you give more details? Please do not set `batch_size` to 1 when evaluating the model. It is not supported in this repo. Sorry.

**Comment by Mulan337287** [comment]
The code is similar to val in main.py

**Comment by Mulan337287** [comment]
Why can not batch_size be 1?
